doc/python_reference.rst

```diff
@@ -293,6 +293,7 @@ Projections:
    compute_proj_raw
    read_proj
    write_proj
+   optical_density
```

Comment:
This is for the `mne` namespace, but it is not imported there. Either move it to the `Preprocessing` section (which covers things that live in the `mne.preprocessing` namespace), or import it to the `mne` namespace via `mne/__init__.py`. This is what CircleCI complains about
```
/home/circleci/project/doc/python_reference.rst.rst:286: WARNING: autosummary: stub file not found 'optical_density'. Check your autosummary_generate setting.
```
My vote would be to move this to the `Preprocessing` section